app/src/androidTest/java/com/github/dedis/student20_pop/MainActivityTest.java

```diff
@@ -47,6 +53,8 @@ public void onClickLaunchTest() {
     @Test
     public void onClickLaunchLAOTest() {
         onView(withId(R.id.tab_launch)).perform(click());
+        onView(withId(R.id.entry_box_launch)).perform(typeText("LAO"));
+        onView(withId(R.id.entry_box_launch)).perform(closeSoftKeyboard());
```

Comment:
you can write this on one line : `.perform(typeText("LAO"), closeSoftKeyboard());`

Reply:
Good tip, thanks